docs/_posts/ahmet-mesut/2024-01-16-sbiobertresolve_snomed_bodyStructure_pipeline_en.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: model
+title: Pipeline for Snomed Concepts and Body Structure Version
+author: John Snow Labs
+name: sbiobertresolve_snomed_bodyStructure_pipeline
+date: 2024-01-16
+tags: [licensed, en, snomed, pipeline, resolver]
+task: [Entity Resolution, Pipeline Healthcare]
+language: en
+edition: Healthcare NLP 5.2.1
+spark_version: 3.2
+supported: true
+annotator: PipelineModel
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
```

Comment:
Update the description with the entity types and what does this pipeline do. ex: https://nlp.johnsnowlabs.com/2023/04/10/medication_resolver_pipeline_en.html